.prettierignore

```diff
@@ -0,0 +1,8 @@
+package.json
+package-lock.json
+yarn.lock
+
+node_modules/
+static/
+
```

Comment:
@mqnguyen5 I think you can also base on .gitignore. Like `/build` and `.docusaurus` you don't want those 2 folders to be reformated

Reply:
Sounds good. I'll try to refer to the `.gitignore` to see what I can add. Thanks!